integration_tests/.scripts/integration_test.sh

```diff
@@ -23,38 +23,44 @@ fi
 
 for db in ${DATABASES[@]}; do
 
-  echo "Snowplow unified integration tests: Running without seeding data"
-  dbt run --full-refresh --target $db
+  # echo "Snowplow unified integration tests: Running without seeding data"
+  # dbt run --select snowplow_unified_base_sessions_lifecycle_manifest --target $db
 
-  status=$?
+  # status=$?
   
-  if [ $status -ne 0 ] && [ $status -ne 1 ]; then
-    echo "dbt command failed for target $db with exit status $status"
-    exit $status
+  # if [ $status -ne 0 ] && [ $status -ne 1 ]; then
+  #   echo "dbt command failed for target $db with exit status $status"
+  #   exit $status
     
-  fi
+  # fi
 
   echo "Snowplow unified integration tests: Seeding data"
-  eval "dbt seed --full-refresh --target $db" || exit 1;
+  eval "dbt seed --full-refresh --target $db --no-partial-parse" || exit 1;
 
   echo "Snowplow unified integration tests: Try run without data"
-  eval "dbt run --full-refresh --vars '{snowplow__allow_refresh: true, snowplow__backfill_limit_days: 1, snowplow__enable_cwv: false, snowplow__start_date: 2010-01-01}' --target $db" || exit 1;
-
+  eval "dbt run --select snowplow_unified_integration_tests.source --full-refresh --vars '{snowplow__allow_refresh: true, snowplow__backfill_limit_days: 1, snowplow__enable_cwv: false, snowplow__start_date: 2010-01-01}' --target $db --no-partial-parse" || exit 1;
+  eval "dbt run --full-refresh --vars '{snowplow__allow_refresh: true, snowplow__backfill_limit_days: 1, snowplow__enable_cwv: false, snowplow__start_date: 2010-01-01}' --target $db --no-partial-parse" || exit 1;
 
   echo "Snowplow unified integration tests: Conversions"
-  eval "dbt run --full-refresh --select +snowplow_unified_conversions snowplow_unified_integration_tests.source --vars '{snowplow__allow_refresh: true, snowplow__backfill_limit_days: 220, snowplow__enable_cwv: false, snowplow__enable_conversions: true}' --target $db" || exit 1;
+  eval "dbt run --full-refresh --select snowplow_unified_integration_tests.source --vars '{snowplow__allow_refresh: true, snowplow__backfill_limit_days: 220, snowplow__enable_cwv: false, snowplow__enable_conversions: true}' --target $db --no-partial-parse" || exit 1;
```

Comment:
I don't think we need to add vars just run the .source bits? (here and below, to reduce extra code duplication)

Reply:
Should we add them so the source files are created "dynamically" correctly with the right conditions?